app-modules/multifactor-authentication/src/Filament/Pages/ManageMultifactorSettings.php

```diff
@@ -46,11 +47,11 @@ class ManageMultifactorSettings extends SettingsPage
 {
     protected static ?string $navigationLabel = 'Multifactor';
 
-    protected static ?int $navigationSort = 50;
```

Comment:
There will be more items in this list within this Sprint, and regardless, all items should have a `navigationSort`. Let's make this one `10`